packages/graphql-server/package.json

```diff
@@ -23,36 +23,33 @@
   },
   "dependencies": {
     "@babel/runtime-corejs3": "7.20.6",
-    "@envelop/depth-limit": "1.8.0",
-    "@envelop/disable-introspection": "3.6.0",
-    "@envelop/filter-operation-type": "3.6.0",
-    "@envelop/parser-cache": "4.7.0",
-    "@envelop/validation-cache": "4.7.0",
+    "@envelop/core": "3.0.4",
+    "@envelop/depth-limit": "2.0.4",
+    "@envelop/disable-introspection": "4.0.4",
+    "@envelop/filter-operation-type": "4.0.4",
     "@graphql-tools/merge": "8.3.12",
     "@graphql-tools/schema": "8.5.1",
     "@graphql-tools/utils": "8.13.1",
-    "@graphql-yoga/common": "2.12.12",
-    "@prisma/client": "4.7.1",
```

Comment:
we don't use either of these (`@graphql-yoga/common`, `@prisma/client`) anymore (`@prisma/client`, not sure if we ever did or when we stopped importing from it)